packages/react-three-rapier/src/utils/utils.ts

```diff
@@ -99,3 +100,13 @@ export function useConst<T>(initialValue: T | (() => T)): T {
   }
   return ref.current.value;
 }
+
+export const vector3ToRapierVector = (v: Vector3ish) => {
```

Comment:
imo we should change all joint hooks to be consistent before releasing so the API surface is consistent. We can do that in a separate PR though.

Reply:
Okay nice thank you ! I was hesitant to extend the changes to all joint hooks until I knew the changes were sound.